crypto/fipsmodule/FIPS.md

```diff
@@ -36,13 +36,7 @@ The utility in `util/fipstools/break-hash.go` can be used to corrupt the FIPS mo
 
 FIPS 140-2 requires that one of its PRNGs be used (which they call DRBGs). In BoringCrypto, we use CTR-DRBG with AES-256 exclusively and `RAND_bytes` (the primary interface for the rest of the system to get random data) takes its output from there.
 
-The DRBG state is kept in a thread-local structure and is seeded from one of the following entropy sources in preference order: RDRAND (on Intel chips), `getrandom`, and `/dev/urandom`. In the case of `/dev/urandom`, in order to ensure that the system has a minimum level of entropy, BoringCrypto polls the kernel until the estimated entropy is at least 256 bits. This is a poor man's version of `getrandom` and we strongly recommend using a kernel recent enough to support the real thing.
```

Comment:
Is it worth keeping any of this to explain what the common passive entropy sources are?

Reply:
There is no "common" source. The source is "unknown" to the fips module. It calls outside the module to obtain entropy.

I don't think we should attempt to store some information here that becomes out-of-sync anyway.

Reply:
Isn't BoringSSL design based on passive entropy as well? If yes, then I think having an idea about where the entropy is drawn from rather than searching in the code or giving the impression that we're not responsible for where the user is drawing entropy from.

Reply:
I don't think this is the right place for such information. It's better to add such documentation adjacent to code. If put here, it will just become obsolete - just like a lot of other information put in markdown files.